blog-posts/step-function-learnings/step-function-logs-learning.md

```diff
@@ -0,0 +1,92 @@
+---
+published: false
+title: "Stop looking for your express state machine logs, they don't exist"
```

Comment:
I think a title more in line with the google search that people will do when facing this exact problem will be more impactful